PathKit/Sources/Angle.swift

```diff
@@ -103,3 +125,15 @@ public func sin(_ angle: Angle) -> Double {
 public func tan(_ angle: Angle) -> Double {
     return tan(angle.radians)
 }
+
+extension Angle: Comparable {
+    public static func < (lhs: Angle, rhs: Angle) -> Bool {
+        return lhs.radians < rhs.radians
```

Comment:
I think this should be normalized:
```suggestion
        return lhs.radians. positiveNormalized < rhs.radians. positiveNormalized
```